examples/kubeval/README.md

```diff
@@ -0,0 +1,32 @@
+# Kubeval
+
+The `kubeval` KRM config function validates configuration using kubeval. This
+example invokes the kubeval function using declarative configuration.
+
+## Function invocation
+
+Get this example and try it out by running the following commands:
+
+```sh
+kpt pkg get https://github.com/GoogleContainerTools/kpt-functions-catalog.git/examples/kubeval .
+kpt fn run kubeval --network --results-dir /tmp
```

Comment:
why does this need to have `--results-dir`? KISS principle: just print to stdout.

Reply:
Removed